netctl/netctl.go

```diff
@@ -245,6 +246,102 @@ func listRules(ctx *cli.Context) {
 	}
 }
 
+func createNetProfile(ctx *cli.Context) {
+	argCheck(1, ctx)
+
+	bandwidth := ctx.String("bandwidth")
+	dscp := ctx.Int("dscp")
+	tenant := ctx.String("tenant")
```

Comment:
if tenant is not mentioned, which is most of the case, it will be set to default.